src/quickdocumentdialog.cpp

```diff
@@ -303,15 +282,65 @@ void QuickDocumentDialog::Init()
         item->setCheckState(Qt::Unchecked);
     }
 
+	QTableWidget *table = ui.tableWidgetPackages;
+	table->clear();
+	table->setColumnCount(2);
+	table->setHorizontalHeaderLabels( QStringList( { tr("Package"), tr("Description") } ) );
```

Comment:
no

```
<item row="1" column="0">
 <widget class="QTableWidget" name="tableWidgetPackages">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
   </sizepolicy>
  </property>
 </widget>
</item>
```

Reply:
shouldn't it be set there ?

Reply:
arraydialog.ui hasn't it (l. 35, obviously because we don't no the columns), bibtexdialog.ui has it (l. 55). So I'll move it, even so I think that it makes translations more complicate.

Reply:
ok, is prepared

Reply:
I was mislead. This doesn't work, reason is calling tableCear() in the init function which is called in each addXxx function. So I returned to my solution.

Reply:
`...->clearContent()`

Reply:
Ah, good hint, that helps. Ok, prepared.